core/src/main/java/org/infinispan/interceptors/TxInterceptor.java

```diff
@@ -129,6 +129,9 @@ public void init(TransactionTable txTable, Configuration configuration, RpcManag
 
    @Override
    public Object visitPrepareCommand(TxInvocationContext ctx, PrepareCommand command) throws Throwable {
+      // Debugging for ISPN-5379
```

Comment:
The whole commit was intended as a commit aid for ISPN-5379, hoping to make the problem more visible and enabling a proper fix.